src/App.js

```diff
@@ -1,21 +1,31 @@
 import React from "react";
+import { Switch, Route, BrowserRouter as Router } from "react-router-dom";
 import "./App.css";
 import { MuiThemeProvider } from "@material-ui/core";
 import CssBaseline from "@material-ui/core/CssBaseline";
 import { theme } from "./theme";
 import SetupPage from "./components/setup/SetupPage";
+import DashboardPage from "./components/dashboard/DashboardPage";
 
 /**
  * The application. Has a theme and routes.
  * @returns {*} The application (JSX)
  */
 function App() {
-  // @TODO: Add routes
   return (
-    <MuiThemeProvider theme={theme}>
-      <CssBaseline />
-      <SetupPage />
-    </MuiThemeProvider>
+    <Router>
+      <MuiThemeProvider theme={theme}>
+        <CssBaseline />
+        <Switch>
+          <Route path="/setup">
```

Comment:
Just out of curiosity, what are you planning on reserving the root `/` path for?

Reply:
That will be a dashboard listing all the hackathons on the site (user might not be logged in)! But now that I think of it, it will be probably the same components as the dashboard, just with different information fed in (such as having log in buttons for the user). I might change this to the root in the future...